### PR TITLE
chore: build spec for aws codebuild

### PR DESCRIPTION
**Pull request by konitroos**
Code build specs for aws codebuild

### CODE REVIEW

File: appspec.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+files:
+  - source: /
+    destination: /app
+    overwrite: true
+
+permissions:
+  - object: /
+    pattern: "**"
+    owner: 219903070214
```
Comment (konitroos) [inline appspec.yml:15]: These ids are for the new instance that is created

File: appspec.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# This is an appspec.yml template file for use with an EC2/On-Premises deployment in CodeDeploy.
+# https://docs.aws.amazon.com/codedeploy/latest/userguide/app-spec-ref.html
```
Comment (konitroos) [inline appspec.yml:2]: https://docs.aws.amazon.com/codedeploy/latest/userguide/app-spec-ref.html

File: scripts/after_install.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# navigate to app folder
+cd /app
```
Comment (konitroos) [inline scripts/after_install.sh:4]: The below three scripts are executed on the EC2 instance

File: scripts/start_server.sh [hunk 4]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# navigate to app folder
+cd app
+
+# initial startup by running react-script "start", name process "singularity_webapp"
+# --watch watches and restarts if files change
+pm2 start ./node_modules/react-scripts/scripts/start.js --name "singularity_webapp" --watch
```
Comment (konitroos) [inline scripts/start_server.sh:8]: More about pm2: https://www.npmjs.com/package/pm2

**Comment by saisumit** [comment]
closing this, migrated to AWS amplify

**Pull request closed without merging**